tests/unit/test-cmd/test.docs.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+import {it, describe} from 'mocha';
+import sinon from 'sinon';
+import {assert} from 'chai';
+
+import docs, {openDocs} from '../../../src/cmd/docs';
+
+
+describe('docs', () => {
+
+  function setUp() {
+    const docsResult = sinon.stub(openDocs, 'openURL');
+    return docsResult;
+  }
+
+  it('passes the correct URL to docs', () => {
+    const docsResult = setUp();
+    const expectedURL = 'https://developer.mozilla.org/en-US/Add-ons' +
+       '/WebExtensions/Getting_started_with_web-ext';
+    docs();
+    assert.equal(docsResult.calledWith((expectedURL)), true);
```

Comment:
I prefer to see an explicit check of the arguments because when it fails it tends to be a little bit easier to diagnose:
````js
assert.ok(docsResult.called);
assert.equal(docsResult.firstCall.args[0], expectedUrl);
````